webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js

```diff
@@ -53,6 +53,15 @@ const ActionsBar = ({
   const deleteHostHandler = () =>
     dispatch(deleteHost(hostId, computeId, destroyVmOnHostDelete));
 
+  const determineToolTip = () => {
+    if (!(computeId && isHostActive)) {
```

Comment:
since we are checking for this condition also below for the disabled state,
can you store that value above, e.g: 
```js
const isConsoleDisabled = !(computeId && isHostActive)
```